lighttpd/datadog_checks/lighttpd/data/conf.yaml.example

```diff
@@ -328,3 +328,26 @@ instances:
     ## This is useful for cluster-level checks.
     #
     # empty_default_hostname: false
+
+## Log Section
+##
+## type - required - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which Integration sent the logs.
+## service - optional - The name of the service that generates the log.
```

Comment:
You may need to rebase and sync due to https://github.com/DataDog/integrations-core/pull/7708 actually

Reply:
Seems like lighttpd didn't have anything added since it didn't have logs at the time, so instead I added the encoding parameter in this PR.